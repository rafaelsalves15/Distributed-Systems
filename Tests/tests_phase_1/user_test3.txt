deleteAccount A alice
transferTo A alice bob 50
deleteAccount A alice
balance A bob
createAccount A alice
transferTo A alice bob 100
createAccount A alice
transferTo A bob bob 100
transferTo A bob alice -10
transferTo A bob alice 0
transferTo A bob charlie 10
transferTo A charlie bob 10
balance A bob
balance A alice
exit
