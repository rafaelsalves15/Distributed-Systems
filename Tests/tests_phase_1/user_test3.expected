UserClientMain
Received 2 arguments
arg[0] = localhost
arg[1] = 2001
> Account not empty
> OK
> OK
> OK
150
> OK
> Insufficient Balance
> Account already exists
> throw an error here (source and dest account are the same)
> throw an error here (amount is negative)
> throw an error here (amount is zero)
> throw an error here (Destination account does not exist)
> throw an error here (Source account does not exist)
> OK
150
> OK
0
